docs/content/configuration/index.md

```diff
@@ -255,6 +255,16 @@ You can enable caching of results at the broker, historical, or realtime level u
 |`druid.cache.maxObjectSize`|Maximum object size in bytes for a Memcached object.|52428800 (50 MB)|
 |`druid.cache.memcachedPrefix`|Key prefix for all keys in Memcached.|druid|
 
+#### Redis
```

Comment:
can we have these docs in a separate file similar to how the other extensions are structured?

Can we also update the extensions.md file to include the new redis extension?